portfolioStyles.css

```diff
@@ -116,6 +134,105 @@ nav img{
 	width: 100%;
 }
 
+.portfolio-header{
+	width: 85%;
+	text-align: center;
+	margin-top: 5%;
+}
+
+.portfolio{
+	width: 85%;
+	height: 900px;
+	padding-top: 12%;
+	overflow: auto;
+	margin-bottom: 70px;
+	text-align: center;
+}
+
+.portfolio div{
+    width: 33.3%;
+    height: 50%;
```

Comment:
Might want to check this height setting as it looks like it may be interfering with the buttons